eng/Subsets.props

```diff
@@ -105,12 +105,13 @@
     <SubsetName Include="Installer" Description="The .NET Core hosts, hosting libraries, bundles, and installers. Includes these projects' tests." />
     <SubsetName Include="CoreHost" Description="The .NET Core hosts." />
     <SubsetName Include="Installer.Managed" Description="The managed .NET hosting projects. This includes HostModel." />
-    <SubsetName Include="Installer.DepProjs" Description="The dependency projects. These gather shared framework files and run crossgen on them to turn them into ready-to-run (R2R) assemblies for the current platform." />
+    <SubsetName Include="nstaller.DepProjs" Description="The dependency projects. These gather shared framework files and run crossgen on them to turn them into ready-to-run (R2R) assemblies for the current platform." />
```

Comment:
Removed `I` 😄

Reply:
Whoops. Don’t know how that happened. When we fully move over to the new SDK (which is the only time this will come out of draft) this subset is getting deleted anyway, so I’ll probably leave this as is for now.

Reply:
Makes sense.